packages/@aws-cdk/aws-apigateway/lib/restapi.ts

```diff
@@ -231,6 +218,25 @@ export class RestApi extends Resource implements IRestApi {
     this.root = new RootResource(this, props, resource.attrRootResourceId);
   }
 
+  /**
+   * API Gateway deployment that represents the latest changes of the API.
+   * This resource will be automatically updated every time the REST API model changes.
+   * This will be undefined if `deploy` is false.
```

Comment:
I don't see how that can be possible from the definition of the private member.

Reply:
Fixed